src/Collection/CollectionBuilder.php

```diff
@@ -139,7 +146,16 @@ public function addTask(TaskInterface $task)
         return $this;
     }
 
-    public function addCode(callable $code, $name)
+  /**
+   * Add arbitrary code to execute as a task.
+   *
+   * @see \Robo\Collection\CollectionInterface::addCode
+   *
+   * @param callable $code
+   * @param int|string $name
+   * @return $this
+   */
+    public function addCode(callable $code, $name = self::UNNAMEDTASK)
```

Comment:
This should use the existing `\Robo\Collection\CollectionInterface::UNNAMEDTASK` rather than defining a new constant. Folks generally won't need to ever put this identifier in their code; they'll just leave the variable off if a name is not desired.